apps/fishing-map/features/dataviews/dataviews.slice.ts

```diff
@@ -176,22 +179,53 @@ export const selectAllDataviewInstancesResolved = createSelector(
   }
 )
 
+export const selectAllDataviewInstancesResolvedWithAppConfig = createSelector(
+  [selectAllDataviewInstancesResolved, selectTimeRange],
+  (dataviewInstances, timerange): UrlDataviewInstance[] | undefined => {
+    if (!dataviewInstances) return
+    return dataviewInstances.map((dataview) => {
+      const isActivityWithContext =
+        dataview.config?.type === GeneratorType.HeatmapAnimated &&
+        dataview.datasetsConfig?.some((d) => d.endpoint === EndpointId.ContextGeojson)
+      if (isActivityWithContext) {
+        return {
+          ...dataview,
+          datasetsConfig: dataview.datasetsConfig?.map((dc) => {
+            if (dc.endpoint === EndpointId.ContextGeojson) {
+              return {
+                ...dc,
+                query: [
```

Comment:
I thought I could do this in the [callbacks generators part](https://github.com/GlobalFishingWatch/frontend/pull/1407/files#diff-1a1503a4e347164a35746c4e098cb27ca8ae3bc8cfdfa12cb989f9c0acfaa3d5R224) but then I realised that configuration is not available at the [resolve-dataviews-generators](https://github.com/GlobalFishingWatch/frontend/pull/1407/files#diff-a4e5867bfb781eaeefaa9225b7f1858d6b69650085b2823f82fafdce6bf2df10R467)

So I needed to add it at the dataview level

Reply:
Because the [resolve-dataviews-generators](https://github.com/GlobalFishingWatch/frontend/pull/1407/files#diff-a4e5867bfb781eaeefaa9225b7f1858d6b69650085b2823f82fafdce6bf2df10R467) handles dataviews instead of resources... so I had to choose between:
- duplicate the logic of adding the params
- adding the configuration in a previous step so it is shared for resources and generators avoiding duplications